src/test/java/net/datafaker/datafaker_gen/DatafakerGenTest.java

```diff
@@ -0,0 +1,57 @@
+package net.datafaker.datafaker_gen;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.of;
+
+class DatafakerGenTest {
+
+    @ParameterizedTest
+    @MethodSource("generateTestParameters")
+    void generateJsonWith10LineAndFileOutput(String input, int expectedLines) throws IOException {
+        DatafakerGen.main(input.split("\\s+"));
+
+        File tempFile = new File("res");
```

Comment:
I guess here it could be used `Files.createTemporaryFile` and then no need for `tempFile.delete();`